crosswalks/BibTeX.csv

```diff
@@ -0,0 +1,74 @@
+Property,BibTeX (@softwareversion)

```

Comment:
I have indicated the entry type like this, does it seem ok?

Reply:
+1